chore: mention npm in docs (#3)
sketch-hq/sketch-assistant-template: README.md

## Testing the Assistant

The Sketch Mac app isn't required to test the Assistant and run the [Jest](https://jestjs.io/)
tests. So a TDD approach using Sketch files as fixtures represents the fastest route to running your
Assistant and checking that it's working as expected.

The tests can be run in watch mode for live feedback as you work.

```
yarn test --watch
```

> ### Test Github Action

Reviewer: It reads a bit oddly, as if I'm testing the GitHub action, maybe something like this:

```suggestion
###  Github Action
```

```suggestion
### Using Github Actions
```

```suggestion
### Testing using Github Actions
```

## Building a local development Assistant

The next quickest way to get the Assistant running is to build it locally and add it directly to a
Sketch document.

1. Run `yarn package-tarball`.
1. Note that a `*.tgz` tarball has been built to the repository root.
1. Use the _Install from disk_ option in Sketch's Assistants menu to add the freshly built Assistant
   to a document.
1. Sketch will check the disk location for updates to the tarball, so you can continue to re-build
   the Assistant to the same location and see live updates in Sketch

> > ⚠️ Sketch documents with local Assistants added to them are not portable. That is, if you share a
> > Sketch document with a local Assistant with a co-worker they will not have access to the same
> > Assistant. If you need a shareable Sketch document with Assistants then you'll need to deploy your
> > Assistant publically. Read the next section for a guide on doing this.

Reviewer: I would inverse the argument, pointing out the benefits of publishing before explaining why the local ones won't work.

```suggestion
> ⚠️ Assistants must be published to be used across multiple devices or within a team. Sketch documents with local Assistants added to them are not portable because the Assistant is referenced using a file path.
```

Since the next paragraph is about publishing, we can ket them just read on.

> ## Deploying the Assistant

Reviewer: ```suggestion
## Publish the Assistant
```

In order for others to use your Assistant via the _Install from URL_ option in Sketch its `*.tgz`
tarball package needs to be hosted publically on the web.

There are obvioulsy many ways to achieve this, but this guide will focus on publishing your
Assistant publically to npm.

1. Review the npm [documentation](https://docs.npmjs.com/packages-and-modules) to familiarize
   yourself about publishing packages, but read on for a quick guide.
1. Ensure you have an npm account and are logged-in to npm on the command line via `npm login`.
1. Update your Assistant code as needed, implement any rules you want to and ensure any tests are
   passing.
1. Ensure the `name` field in package.json has been set to a valid, unique npm package name.

> 1. Ensure the `version` field in package.json is updated to reflect the version you want to publish.

Author: The keywords are already in the template's package.json ... so that side of things is handled I think?

https://github.com/sketch-hq/sketch-assistant-template/blob/master/package.json#L13-L19
